internal/web_api.go

```diff
@@ -178,3 +181,10 @@ func DefaultHTTPRequestGenerator(name string) HTTPRequestGenerator {
 		return http.NewRequest("POST", url, nil)
 	}
 }
+
+func (a *api) tokenTypeToString(tokenType authorization.TokenType) string {
+	if tokenType == authorization.BasicToken {
+		return "Basic"
+	}
+	return "Bearer"
+}
```

Comment:
please make it a `String()` method with `TokenType` as receiver, then you can simply pass it to `fmt.Sprintf`